pkgs/development/libraries/gdal/default.nix

```diff
@@ -3,7 +3,8 @@
 , libpng
 , netcdf, hdf5 , curl
 , netcdfSupport ? true
- }:
+, pythonSupport ? !stdenv.isDarwin
+}:
```

Comment:
actually, that won't make a difference, because we override the expression. You can ignore my previous comment. :-)